rpmbuild/copr-rpmbuild.spec

```diff
@@ -40,9 +40,14 @@ BuildRequires: %{python_pfx}-munch
 BuildRequires: %{python}-requests
 BuildRequires: %{python_pfx}-jinja2
 BuildRequires: %{python_pfx}-simplejson
-BuildRequires: %{python_pfx}-specfile
 BuildRequires: python3-backoff >= 1.9.0
 
+%if 0%{?rhel} == 8
+BuildRequires: %{python_pfx}-specfile
```

Comment:
I submited the issue https://github.com/packit/specfile/issues/255 for this.

Reply:
I tested a bit specfile v0.20 on epel8 and wasn't able to make it work. So I don't think there's an easy way to update specfile into EPEL8 (missing setup.py, incompatible (too new) typing, I stopped there...).

The question is - is the python-specfile in epel 8 "new enough" for us (100% working, but slow), or does this actually mean we have to drop the support for EPEL8?

Reply:
Getting to EPEL8 https://bodhi.fedoraproject.org/updates/FEDORA-EPEL-2023-8a36e6e848 so we can probably drop the %if/%else